treewide: nuke packages in favour of wallpapers

amadaluzia opened the pull request:
## What does this do?
This PR aims to change how the wallpaper overlay gets all of its packages.

## Why?
I find this may adding wallpaper directories that one little bit more braindead simple.
Additionaly, the `rec` statement could be removed by moving wallpkgs to the `let in` expression.

amadalusia committed:
refactor(flake): remaster wallpapers

amadalusia committed:
feat(nord): init nord wallpapers

NotAShelf reviewed (approved):
LGTM.

Ready for merge or do we want to add something to generate the list of wallpapers?

amadaluzia commented:
I feel as if it would be best to carry a Markdown list generator in another PR.

NotAShelf commented:
Okay sounds good. If you could just add a warning to tell users about what to use instead when `packages.*` is accessed, that'd be great. For now it'll silently error, and we shouldn't let that happen.

amadalusia committed:
feat!(packages): make packages throw an error

We have refactored the wallpaper system to be similar to what [yunfachi
did with their wallpapers](https://github.com/yunfachi/nix-config-wallpapers).

This in turn, had made packaging obsolete and it was decided that it
would be removed.

NotAShelf commented:
thank you

yunfachi commented on flake.nix:
```diff
@@ -9,46 +9,113 @@
+        "nature-02" = ["nature"];
+        "nature-03" = ["nature"];
+        "nature-04" = ["nature"];
+        "nature-05" = ["nature"];
+        "nature-06" = ["nature"];
+        "nature-07" = ["nature"];
+        "oxocarbon-01" = ["oxocarbon"];
+        "oxocarbon-02" = ["oxocarbon"];
+        "rose_pine-01" = ["rose_pine"];
+        "rose_pine-02" = ["rose_pine"];
+        "rose_pine-03" = ["rose_pine"];
+        "rose_pine-04" = ["rose_pine"];
+        "rose_pine-05" = ["rose_pine"];
+        "rose_pine-06" = ["rose_pine"];
+        "space-01" = ["space"];
+        "space-02" = ["space"];
+        "space-03" = ["space"];
+        "space-04" = ["space"];
+        "tokyo_night-01" = ["tokyo_night"];
+        "tokyo_night-02" = ["tokyo_night"];
+        "tokyo_night-03" = ["tokyo_night"];
+        "tokyo_night-04" = ["tokyo_night"];
+        "tokyo_night-05" = ["tokyo_night"];
+        "unorganized-01" = ["unorganized"];
+        "unorganized-02" = ["unorganized"];
+        "unorganized-03" = ["unorganized"];
+        "unorganized-04" = ["unorganized"];
+        "unorganized-05" = ["unorganized"];
+        "unorganized-06" = ["unorganized"];
+        "unorganized-07" = ["unorganized"];
+        "unorganized-08" = ["unorganized"];
+        "unorganized-09" = ["unorganized"];
+        "unorganized-10" = ["unorganized"];
+        "unorganized-11" = ["unorganized"];
+        "nord-01" = ["nord"];
+        "nord-02" = ["nord"];
+        "nord-03" = ["nord"];
+        "nord-04" = ["nord"];
+        "nord-05" = ["nord"];
+      };
```
@amadaluzia, Awesome Work! **But, you know, giving credit where it's due is just basic respect and decency.** Open source isn't a free pass to copy-paste and call it your own. **I’d hope you understand how it feels, but I really doubt you have ever produced a single original thought. May you have the day you deserve.** Just changing "fileExtensions" to "fileExts" and swapping != for == is a bit boring.
<sub>© NotAShelf [meta: archive repository](https://github.com/NotAShelf/nyx/commit/d407b4d6e5ab7f60350af61a3d73a62a5e9ac660)</sub>

Written by me on September 7, 2024: https://github.com/yunfachi/nix-config-wallpapers/blob/b346206252b2937f2308b08332a0898c9230b2bd/default.nix

@NotAShelf, Please credit my repository, **This is not
the spirit of open source.**

NotAShelf replied:
I understand that you are frustrated, and you are right to be but harassing my contributors and nitpicking (apparrently, also mocking) my words to do so is not the way to go here. This could have easily be resolved privately, you could have e-mailed me and it'd be over. It was me who introduced your repository to them as a point of reference,and chances are they *simply forgot*.

It's late, and my computer is off. I'm not going to be able to do anything now, and nor do I feel compelled to handle the situation urgently. Your feelings are *not* discarded, as I agree with you, but I also don't agree with the way you chose to engage. If I do end up forgetting, please do feel free to remind me. 

Cheers.

yunfachi replied:
Ok, thanks
P.S. it's a joke, sorry!

amadaluzia replied:
Hey there! Yeah, seems I did forget to credit. I did end up changing a LOT of how it works behind the scenes in post. See the latest version of the flake. ^^

I apologise for forgetting to credit, I'll make a PR for that now.
